packages/@cruise-automation/hooks/babel.config.js

```diff
@@ -4,9 +4,6 @@
 //  found in the LICENSE file in the root directory of this source tree.
 //  You may not use this file except in compliance with the License.
 
-const baseConfig = require("../../../babel.config");
+const baseConfig = require("../../regl-worldview/babel.config");
```

Comment:
Should we pull this out to a different directory, instead of pulling files from a sibling directory? That can be pretty confusing after all.